napari_allencell_annotator/controller/annotator_controller.py

```diff
@@ -86,7 +90,8 @@ def start_annotating(self, num_images: int):
         num_images : int
             The total number of images to be annotated.
         """
-
+        for path, lst in dct.items():
```

Comment:
You're unpacking `dct` to create the same dictionary `self.annotation_dict`. You can just do `self.annotation_dct = dct`